### ISSUE TITLE
Improve documentation of what has been "done" to the data returned by readers

### ISSUE THREAD

**Issue by BENR0** [pytroll/satpy#1694]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Lately I have been asked by members of our working group working with the Seviri readers about what is actually "done" to the data by the readers (e.g. what corrections are applied, are quality flags checked). I think this are valid questions and while, specifically for the Seviri readers, some information can be found in the documentation it is rather hard to find. I think it would be good if this information is accessible more easily. 

**Describe the solution you'd like**
My proposal would be that this should be "prominently" in the documentation of the readers themselves and maybe this is something that also can be integrated in the reader table for a quick overview.

**Describe any changes to existing user workflow**
No changes.

**Comment by djhoese** [comment]
Great idea! I think this has been discussed in other issues so this may be a duplicate, but I wanted to say I completely agree.

**Comment by djhoese** [comment]
I think we need to come up with a kind of "standard" series of sections that should be included in reader module docstrings (calibration, navigation, etc) that explain the details of what's going on.

**Comment by BENR0** [comment]
yes that would be ideal.

**Comment by sfinkens** [comment]
Good idea! I think it would also be nice to "document" it in the dataset attribute. See for example the `sun_earth_distance_corrected` attribute.
